app.json

```diff
@@ -63,7 +63,7 @@
     },
     "DISABLE_COLLECTSTATIC": {
       "description": "Disables running ``./manage.py collectstatic`` during the build. Should be set to ``1``.",
-      "value": "1"
+      "value": ""
```

Comment:
We also need to add `"required": false,` here.

By default the field will be required and without a default value it's not possible to just "Deploy app" with a click of a button.

Reply:
Added. This part of the build still feels a bit too mysterious to me.